pkg/batch/jobs/rediscounter/redis_counter_deleter_test.go

```diff
@@ -307,19 +307,11 @@ func TestFilterKeysOlderThanThirtyOneDays(t *testing.T) {
 		expected          []string
 		expectedError     error
 	}{
-		{
-			desc:              "errParseInt",
-			inputEnvNamespace: "dev",
-			inputKind:         "uc",
-			inputKeys:         []string{"dev:uc:timestamp:feature_id:variation_id"},
-			expected:          nil,
-			expectedError:     errParseInt,
-		},
```

Comment:
Since we updated the regex to match timestamp only, we don't need this test anymore.